ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -153,28 +150,25 @@ export class WorkspaceCard extends React.Component<WorkspaceCardProps, Workspace
   constructor(props) {
     super(props);
     this.state = {
-      bugReportError: '',
-      bugReportOpen: false,
       confirmDeleting: false,
       loadingData: false,
       showShareModal: false,
       showResearchPurposeReviewModal: false,
     };
   }
 
-  async deleteWorkspace() {
-    const {workspace} = this.props;
-    this.setState({
-      confirmDeleting: false,
-      loadingData: true});
-    try {
-      await workspacesApi().deleteWorkspace(workspace.namespace, workspace.id);
-      this.setState({loadingData: false});
```

Comment:
looks like this state is no longer set.  Does this mean that there is no longer a spinner here?  @aweng98

Reply:
I saw spinner during my local testing. Did you think this is wrong or see some weird behavior?

Reply:
No, everything looks fine.  We can actually remove this now: https://github.com/all-of-us/workbench/pull/5218